pkg/cmd/container/cp_linux.go

```diff
@@ -0,0 +1,35 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package container
+
+import (
+	"context"
+
+	"github.com/containerd/nerdctl/pkg/api/types"
+	"github.com/containerd/nerdctl/pkg/containerutil"
+)
+
+// Cp copies files/folders between a running container and the local filesystem.
+func Cp(ctx context.Context, options types.ContainerCpOptions) error {
+	return containerutil.CopyFiles(
+		ctx,
+		options.Container2Host,
+		options.Pid,
+		options.DestPath,
+		options.SrcPath,
+		options.FollowSymLink)
+}
```

Comment:
Personally I don't feel it's necessary to have this wrapper (i.e., cmd can call `containerutil.CopyFiles` directly), although it makes it consistent with other commands.

But you can keep it unless other reviewers also prefer not having this wrapper :)

Reply:
Yes I was also hesitant to do the additional wrapper, just like I said:
 > This way the final logic will remain very similar to the origin ── just an additional Options and the separated Kopy func, which is redundant?
> Maybe I should just close the PR and check the cp refactoring box?

But I end up keeping it because I think the consistent workflow of cmd processing may provide better maintainability for further development.
@AkihiroSuda Pls give your valuable comments. 🙏🏻

@djdongjin Again, thank you very much for your patience and valuable comments.
This is the first time I've submitted a PR to the open source community and you guys really have given me confidence. ❤️️